server/http_tunnel.go

```diff
@@ -18,7 +18,11 @@ import (
 
 // StartHTTPTunnel starts a server that accepts to the HTTP CONNECT method to proxy arbitrary TCP connections.
 // It can be used to tunnel HTTPS connections.
-func StartHTTPTunnel(addr string, externalProxy string) {
+func StartHTTPTunnel(addr, externalProxy string) {
+	StartHTTPSTunnel(addr, externalProxy, "", "")
+}
+
+func StartHTTPSTunnel(addr, externalProxy, certFile, KeyFile string) {
 	log.Printf("Starting HTTPS proxy on %s\n", addr)
```

Comment:
This message is bound to cause confusion now. It would serve much better if it were down in the `if` statement, printing slightly different thing depending on the branch (TLS vs plain).